static/app/views/issueList/actions/index.tsx

```diff
@@ -47,6 +48,13 @@ type IssueListActionsProps = {
   onActionTaken?: (itemIds: string[], data: IssueUpdateData) => void;
 };
 
+const animationProps = {
```

Comment:
There's a type you can set on this. I also usually just slap these on the defaultProps of the styled component.